source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,247 @@
+
+.. tip:: Supported Operators
+
+   Spherical (``2dsphere``) and flat (``2d``) indexes support some, but
+   not all, of the same query operators. For a full list of operators
+   and their index compatibility, see the
+   :manual:`manual entry for geospatial queries </geospatial-queries/#geospatial-query-operators>`.
+
+   .. external resource
+
+Geospatial Queries
+------------------
+
+To perform a geospatial query, create a query filter with a field name and a geospatial
+query operator. You can specify additional options for certain geospatial query operators
+to limit the documents returned.
+
+Query Operators
+~~~~~~~~~~~~~~~
+
+To query your geospatial data, use one of the following query operators: 
+
+- ``$near``
+- ``$geoWithin``
+- ``$nearSphere``
+- ``$geoIntersects`` *requires a 2dsphere index*
+
+When using the ``$geoWithin`` operator, you can specify the following shape operators:
+
+- ``$box``
+- ``$polygon``
+- ``$center``
+- ``$centerSphere``
+
+When using the ``$near`` operator, you can specify the following distance operators:
+
+- ``$minDistance``
+- ``$maxDistance``
+
+For more information on geospatial query operators, see the
+:manual:`manual entry for geospatial queries </geospatial-queries/#geospatial-query-operators>`.
+
+.. external resource
+
+Examples
+--------
+
+The following examples use the MongoDB Atlas sample dataset. You can load sample datasets into 
+your database on the free tier of MongoDB Atlas by following the :atlas:`Get Started with Atlas Guide
+</getting-started/#atlas-getting-started>` or you can :guides:`import the sample dataset into a local MongoDB instance
+</server/import/>`.
+
+The examples use the ``theaters`` collection in the ``sample_mflix`` database
+from the sample dataset. The ``theaters`` collection contains a ``2dsphere`` index
+on the ``location.geo`` field.
+
+Query by Proximity
+~~~~~~~~~~~~~~~~~~
+
+Code example:
+
+.. code-block:: go
+
+   location := bson.D{{"type", "Point"}, {"coordinates", []float32{-76.512016, 38.29697}}}
+
+   filter := bson.D{
+      {"location.geo",
+         bson.D{
+            {"$near", bson.D{
+            {"$geometry", location},
+            }},
+         }},
+   }
+   var places []bson.D
+   output, err := coll.Find(context.TODO(), filter)
+   if err = output.All(context.TODO(), &places); err != nil {
+      panic(err)
+   }
+   fmt.Println(places)
+
+Query Within a Range
+~~~~~~~~~~~~~~~~~~~~
+
+Code example:
+
+.. code-block:: go
+
+   location := bson.D{{"type", "Point"}, {"coordinates", []float32{-73.9667, 40.78}}}
+
+   filter := bson.D{
+      {"location.geo",
+         bson.D{
+            {"$nearSphere", bson.D{
+            {"$geometry", location},
+            {"$maxDistance", 10000},
+            }},
+         }},
+   }
+   var places bson.D
+   err := coll.FindOne(context.TODO(), filter).Decode(&places)
+   if err != nil {
+      panic(err)
+   }
+   fmt.Println(places)
+
+Additional Resources
+--------------------
```

Comment:
Issue: There's nothing here.

Suggestion: Add additional resources or delete this heading.